app/controllers/api/test_results_controller.rb

```diff
@@ -134,9 +134,7 @@ def update
 
     # User params for create & update
     def test_result_params
-      params.permit(:name, :completion_status, :marks_earned,
-                    :input, :actual_output, :expected_output, :created_at,
-                    :updated_at)
+      params.permit(:name, :status, :marks_earned, :output, :created_at, :updated_at)
```

Comment:
Happened to notice the `:created_at` and `:updated_at`. They'd only be necessary if the autotesting server were passing these.

Reply:
You're right, didn't think about that.